Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/TodoConfiguration.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftLintCore
+
+enum TodoKeyword: String, CaseIterable, AcceptableByConfigurationElement {
```

Comment:
Nit: We typically have such enums as inner types inside of the configuration struct type.